### PR TITLE
PCQ Production front-door config and lb config

### PR DESCRIPTION
**Pull request by ManojKulkarni-HMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/secure/RapidBoard.jspa?rapidView=1305&view=detail&selectedIssue=PCQ-271

### Change description ###
Added front-door config and load balancer config for PCQ project in production environment.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: environments/prod/prod.tfvars [hunk 1]
```diff
@@ -916,5 +916,13 @@ frontends = [
...
     ]
-  }
+  },
+  {
+    name             = "pcq"
+    mode             = "Detection"
+    custom_domain    = "equality-and-diversity.platform.hmcts.net"
+    backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
+    certificate_name = "equality-and-diversity-platform-hmcts-net"
+    www_redirect     = true
```
Comment (adusumillipraveen) [inline environments/prod/prod.tfvars:926]: Is this intentional ?

Comment (adusumillipraveen) [reply environments/prod/prod.tfvars:926]: I am guessing it's not as i cannot see dns entry for it.

Comment (adusumillipraveen) [inline environments/prod/prod.tfvars:926]:
```suggestion
```

Comment (ManojKulkarni-HMCTS) [reply environments/prod/prod.tfvars]: This has been removed now.

Comment (ManojKulkarni-HMCTS) [reply environments/prod/prod.tfvars]: No, we don't. www_redirect has been removed.

**Pull request merged**